simd.dd

```diff
@@ -92,8 +100,8 @@ $(H3 Conversions)
 	to their vector equivalents:)
 
 ---
-int4 v = 7;
```

Comment:
What's the recommended way to multiply a float vector by a constant using SIMD then? I mean a compile ought to be able to replicate a vector of 3's and do the normal vector x vector SIMD multiply.